test/Game-test.js

```diff
@@ -0,0 +1,78 @@
+const chai = require('chai');
+const expect = chai.expect;
+const Game = require('../js/Game.js');
+const spies = require('chai-spies');
+chai.use(spies);
+global.data = require('../js/data.js');
+global.domUpdates = require('../js/DOM.js');
+global.Round = require('../js/Round.js');
+// global.index = require('../js/index.js');
+chai.spy.on(global.domUpdates, ['getPlayerNames', 'clearInputs', 'goToGameScreen', 'displayWinner', 'goToHomeScreen', 'displayWheel', 'hideWheel'], () => true);
+
+describe('Game', () => {
+  var game;
+  beforeEach(() => {
+    game = new Game()
+  })
+
+  it('should start at round 0', () => {
+  expect(game.round).to.equal(0);
+  expect(game.bonusRound).to.equal(false);
+  });
+
+  it('should start with no players', () => {
+  expect(game.players).to.deep.equal({});
+  });
+
+  it('should start with 4 puzzle banks', () => {
+    expect(game.puzzleKeys).to.deep.equal(['one_word_answers', 'two_word_answers', 'three_word_answers', 'four_word_answers']);
+  });
+
+  it.skip('should be able to take in player names', () => {
+   
+  });
+
+  it('should start a new game', () => {
+    game.init();
+    expect(global.domUpdates.clearInputs).to.have.been.called(1);
+    expect(global.domUpdates.goToGameScreen).to.have.been.called(1);
+  });
+
+  it('should be able to start a new round', () => {
+    game.startRound();
+    expect(game.round).to.equal(1);
+  });
```

Comment:
Also vague here, same concept as above - wrap this in a describe block specifically for starting a new round, and tell me exactly what should happen when a new round is started.